sldns/keyraw.c

```diff
@@ -428,7 +428,7 @@ sldns_key_buf2rsa_raw(unsigned char* key, size_t len)
 		BN_free(modulus);
 		return NULL;
 	}
-#if OPENSSL_VERSION_NUMBER < 0x10100000 || defined(HAVE_LIBRESSL)
+#if OPENSSL_VERSION_NUMBER < 0x10100000
```

Comment:
Would the following make sense to keep current behavior for libressl < 3.5 ?
```suggestion
#if OPENSSL_VERSION_NUMBER < 0x10100000 || \
        (defined(HAVE_LIBRESSL) && LIBRESSL_VERSION_NUMBER < 0x03050000f)
```

Reply:
Yes, that would work, too.
Theo Buehler (tb@openbsd) tells me that `LIBRESSL_VERSION_NUMBER < 0x2070000f` would be even better.

Reply:
I guess he should know :)